cookbook/email/dev_environment.rst

```diff
@@ -119,6 +119,58 @@ the replaced address, so you can still see who it would have been sent to.
     These are ``X-Swift-Cc`` and ``X-Swift-Bcc`` for the ``CC`` and ``BCC``
     addresses respectively.
 
+Sending to a Specified Address, but with exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Suppose you normally in your dev environment want to have all email redirected
+to a specific address, (like in the above scenario to ``dev@example,com``).
+But then you may want email sent to some specific email addresses to go through
+after all, and not be redirected (even if it is in the dev environment).
+This can be done by adding the ``delivery_whitelist`` option:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config_dev.yml
+        swiftmailer:
+            delivery_address: dev@example.com
+            delivery_whitelist:
+               # all email addresses matching this regex will *not* be
```

Comment:
I don't know which is the common practice in the Symfony docs (maybe @WouterJ and @xabbuh can help us with this) but it's strange to add explanation comments in the PHP format but not include them in the XML and PHP formats. We have two solutions:

  * Repeat the same explanation for all formats.
  * Don't add these comments and just explain it using the regular text content.